soroban-env-host/src/budget.rs

```diff
@@ -294,79 +294,79 @@ impl Default for BudgetImpl {
                     cpu.lin_term = ScaledU64::from_unscaled_u64(1).safe_div(8);
                 }
                 ContractCostType::DispatchHostFunction => {
-                    cpu.const_term = 310;
+                    cpu.const_term = 295;
```

Comment:
While this seems _relatively_ safe, I'm not sure if changing the hardcoded values is the way forward. Probably env should just settle on some calibration and then SDK could be updated to match the network settings.